doc/user_manual/generated/sklRom.tex

```diff
@@ -35,6 +35,163 @@ \subsubsection{LinearDiscriminantAnalysisClassifier}
       etc) used in the input HistorySet.
   \default{time}
 
+    \item \xmlNode{featureSelection}:
+      Apply feature selection algorithm
+
+      The \xmlNode{featureSelection} node recognizes the following subnodes:
+      \begin{itemize}
+        \item \xmlNode{RFE}:
+          The \xmlString{RFE} (Recursive Feature Elimination) is a feature selection algorithm.
+          Feature selection refers to techniques that select a subset of the most relevant features
+          for a model (ROM).         Fewer features can allow ROMs to run more efficiently (less
+          space or time complexity) and be more effective.         Indeed, some ROMs (machine
+          learning algorithms) can be misled by irrelevant input features, resulting in worse
```

Comment:
Is it possible to add featureSelection part in the ROM section, instead of adding for each individual ROM?

Reply:
This comes from the automatic script. I can do it manually but it will be "overwritten" by the next person "re-creating" the documentation. @wangcj05 please advice on how to proceed

Reply:
I see, it is fine to me as it is.

Reply:
@wangcj05 I opened an issue to capture this 

https://github.com/idaholab/raven/issues/2075